cekit/templates/template.jinja

```diff
@@ -32,16 +32,16 @@ COPY \ {% for repo in packages.repositories_injected %}
 {% if packages.repositories %}
 {% for repo in packages.repositories %}
 {% if repo.present and repo.rpm %}
-RUN yum install -y {{ repo.rpm }} \
-    && yum clean all && rm -rf /var/cache/yum
+RUN {{ pkg_manager }} install -y {{ repo.rpm }} \
```

Comment:
This seems a bit fragile, as it assumes the subcommands are the same between package managers.  Perhaps it might be better to stub this out into separate macros that are invoked based on the specified package manager, e.g. `{{ dnf_install(packages) }}' or `{{ yum_install(packages) }}`.

Reply:
This is true, but I'd argue not a blocker. The commands are common at least for yum, dnf and microdnf